### PR TITLE
refpolicy: Escape meta characters

### PR DESCRIPTION
**Pull request by crogers1**
Currently, some .fc refpolicy files are using the meta character
  . literally, in the case of an absolute path to a file with an
  extension (for example ".conf"). These meta characters should
  be escaped to avoid a potentional mismatch.

  OXT-1803

Signed-off-by: Chris Rogers <rogersc@ainfosec.com>

### CODE REVIEW

File: recipes-security/refpolicy/refpolicy-mcs-2.%/policy/modules/services/vusb.fc [hunk 1]
```diff
@@ -21,5 +21,5 @@
 /usr/bin/usbowls        --      gen_context(system_u:object_r:vusbc_exec_t,s0)
 /usr/sbin/vusb-daemon	--	gen_context(system_u:object_r:vusbd_exec_t,s0)
 /etc/USB_fakenames.conf	--	gen_context(system_u:object_r:vusbd_etc_t,s0)
```
Comment (eric-ch) [inline recipes-security/refpolicy/refpolicy-mcs-2.%/policy/modules/services/vusb.fc:23]: `/etc/USB_fakenames.conf` as well :).

**Review by dpsmith — APPROVED** [review]

File: recipes-security/refpolicy/refpolicy-mcs-2.%/policy/modules/services/vusb.fc (hunk 1 above)
Comment (crogers1) [reply recipes-security/refpolicy/refpolicy-mcs-2.%/policy/modules/services/vusb.fc:23]: oof good catch.

**Comment by eric-ch** [comment]
There are a few more in the patches, do you mind addressing these in this PR @crogers1 ?
- https://github.com/OpenXT/xenclient-oe/blob/master/recipes-security/refpolicy/refpolicy-mcs-2.%25/patches/policy.modules.system.xen.diff#L7
- https://github.com/OpenXT/xenclient-oe/blob/master/recipes-security/refpolicy/refpolicy-mcs-2.%25/patches/policy.modules.system.xen.diff#L14
- https://github.com/OpenXT/xenclient-oe/blob/master/recipes-security/refpolicy/refpolicy-mcs-2.%25/patches/policy.modules.system.authlogin.diff#L16
- https://github.com/OpenXT/xenclient-oe/blob/master/recipes-security/refpolicy/refpolicy-mcs-2.%25/patches/policy.modules.admin.brctl.diff#L7
- https://github.com/OpenXT/xenclient-oe/blob/master/recipes-security/refpolicy/refpolicy-mcs-2.%25/patches/policy.modules.admin.alsa.diff#L12

**Comment by crogers1** [comment]
Yea I can take care of those too @eric-ch

**Review by jandryuk — APPROVED** [review]
Thanks @crogers1 .  Did one bite you, or did you just notice it when working on the policy?

**Comment by crogers1** [comment]
Troy noticed it awhile ago and I'm just finally getting some time to do cleanup/backlog stuff.  But nope, we never observed any runtime issues.

**Comment by jandryuk** [comment]
Got it.  Thanks.

**Comment by eric-ch** [comment]
[Build](https://openxt.ainfosec.com/buildbot/#/builders/14/builds/14): [$mcql-14](https://openxt.ainfosec.com/OpenXT/auto/master/master-custom-quick-legacy/14/)

**Review by eric-ch — APPROVED** [review]
Will merge soon.

**Pull request merged**